viite-UI/src/view/RoadAddressBrowserWindow.js

```diff
@@ -221,8 +221,8 @@
                 $('#exportAsExcelFile').prop("disabled", false); // enable Excel download button
             }
             else {
-                // hide the results and notify user to download result table as excel file
-                roadAddrBrowserWindow.append($('<p id="tableNotification"><b>Tulostaulu liian suuri, lataa tulokset excel taulukkona</b></p>'));
+                // hide the results and notify user to download result table as Excel file
```

Comment:
Nil: kieli
// hide the results_,_ and notify _the_ user to download _the_ result table as _an_ Excel file

...uh, englannin kieli koostuu pelkistä artikkeleista  :D